### PR TITLE
Hero section

### PR DESCRIPTION
**Pull request by charlesgobina**
For this milestone, I
- Created the home and about page of a music concert site
- I styled both pages to match the design as close as possible

### Fixes
* Fixed the session height issue on the index and about pages
* Fixed close button on mobile nav and made it appear far right
* Added home tab to the navbar
* Fixed live link
* [Loom Video](https://www.loom.com/share/696d32e9eafc48b5ae46e6c7f480438c)

### Fixes
* Fix main program area
* Add footer-logo as required

## Fixes
* Fix close button on about page

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+      </div>
+      <nav class="d-flex">
+        <a class="burger" href="#">
+          <i class='bx bx-menu bx-md'></i>
+        </a>
+        <a href="index.html">
+          <h1 class="frank-logo" >Musica</h1>
+        </a>
+        <ul class="desktop-links">
+          <li class="primary-color"><a href="about.html">About</a></li>
```
Comment (dasileker) [inline about.html:33]: - [ ] The about page nav should contain a home link page.

nav

File: about.html [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        <div class="close-btn">
+          <i class="bx bx-x bx-lg closeIcon"></i>
+        </div>
+        <ul>
+          <li><a class="side-link" href="index.html">Home</a></li>
+          <li><a class="side-link" href="about.html">About</a></li>
+          <li><a class="side-link" href="#aboutme">Program</a></li>
+          <li><a class="side-link" href="#contact">Join</a></li>
+          <li><a class="side-link" href="#contact">Sponsor</a></li>
+          <li><a class="side-link" href="#contact">Campaign</a></li>
```
Comment (dasileker) [inline about.html:51]: - [ ] The mobile menu also, remove the about page link. and re-adjust the margin of the links and the `X` close button to the far right.

m

File: about.html [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+  </div>
+  <main>
+    <section class="about">
+      <h4 class="program-text secondary-color">Franky Global Musical Concert 2022 logo</h4>
+      <hr class="primary-color line">
+      <p class="hero-text contact-text">
+        This Logo was designed by Graphixray_Designz back at the February 2016 summit. It was accepted after winning
+        an award of best logo design. Graphixray_Designz was given a noble prize.
+      </p>
+      <div class="concert-logo">
```
Comment (dasileker) [inline about.html:82]: - [ ] The section is under, the div please re-adjust the CSS.

section

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @charlesgobina,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- As the requirements say, you should record a 5 min video in which you should explain your project.
- And also your Live Demo link is not working correctly, and also add an image of your project in your readme file.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 1 above)
Comment (DeliceLydia) [reply about.html:33]: Addressed this

File: about.html (hunk 3 above)
Comment (DeliceLydia) [reply about.html:82]: - [ ] Addressed this but the footer should contain a logo aligned to the left and text aligned to the right.
check the original
footer

File: index.html [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+        </div>
+      </div>
+      
+    </section>
+    <section class="feature-artists">
+      <h4 class="program-text secondary-color">Team Members</h4>
+      <hr class="primary-color line">
+      <div class="program-details artists">
+        
+      </div>
```
Comment (DeliceLydia) [inline index.html]: ## Optional
- [ ] I highly suggest that you work on this section
 this is the original
mai

your page
org

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (charlesgobina) [reply index.html]: Hi @DeliceLydia 
Should I use the same icons?
Should I also add more events?

Comment (DeliceLydia) [reply index.html]: Hello @charlesgobina,
Actually, the requirements said that you should replicate the design
only contents you are allowed to personalize. Icons
you can use yours but events can be the same number, background, colors, fonts should be the same

**Review by VanessaAoki — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED :yellow_circle: 
Hi @charlesgobina,

Great work on making the changes requested by the previous reviewers, but there is still one more which isn't addressed yet.


## Required Changes ♻️
- [ ] When in the mobile version, if you open the menu on the About page, your `x` button to close the menu is still misplaced, you should consider moving it to the right, to match the Home page.

How it looks in the home page:
image


How it looks in the about page:
image



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


really close

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## Thank you for implementing all the project requirements, you've done a great job :trophy: There is nothing else to say other than... it's time to merge it :champagne:  :tada: Congratulations

Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**